### DISCUSSION TITLE
VSC Collaborative Editing not Working on Guests (v1.0.5883)

### DISCUSSION THREAD

**Discussion by RougeSpaceWizzard** [microsoft/vscode-discussions#890] [Extension Development QnA] [answered]
As Part of a Project in Vocational school, I have to work together with multiple Classmates. We decided to work together in VS Code and use the Collaborative Editing Feature. We quickly realized, that the Guests didn't have the intellisense Features enabled, and could only manually type the Text, with no Autocompletion.

The Host of the Session has full access to these features, as seen in the Screenshot

![image](https://github.com/microsoft/vscode-discussions/assets/145323586/78959b1f-34ae-4d0e-8822-abc016b47fc3)
The Clients on the other hand have no such features, and can only write raw Text. Since we all use different Devices and work for different Companies, I unfortunately cannot attach any of their Log Files.

![image](https://github.com/microsoft/vscode-discussions/assets/145323586/1bce7766-a9e0-4a21-a6fb-0f0d78625437)

This happens on both the Web, as well as the App Version of VSCode!
I researched this on multiple forums. I read that this feature broke in the most recent update, and that you would have to revert to a previous Version to fix this. These posts were older than 2 Years, so I don't think, they are up do Date anymore. Is there a Setting, that is turned off by Default, or do we have to revert to a older Version to fix this Problem.

Thanks to all of you
**Your RSW,**

**Comment by olguzzar** [comment, marked as answer]
Hi @RougeSpaceWizzard, I assume you were using the Live Share extension. If so, you should head over to the Live Share repo and check out the existing issues - it looks like [this one](https://github.com/MicrosoftDocs/live-share/issues/4893) is currently active and similar to the problem you are having. You can comment on that issue with additional information to help the developers troubleshoot. 

This forum is for extension authors, so I will be closing this discussion. Thanks!
